src/Web/Sfa.Das.Sas.Infrastructure/ElasticSearch/ElasticsearchProviderLocationSearchProvider.cs

```diff
@@ -272,7 +275,8 @@ private SearchDescriptor<T> CreateProviderQueryWithoutLocationLimit<T>(Expressio
                     .Query(q => q
                         .Bool(b => b
                             .Filter(FilterByApprenticeshipId(selector, code))
-                            .Must(NestedLocationsQueryWithoutLocationMatch<T>(location))))
+                            .Must(NestedLocationsQueryWithoutLocationMatch<T>(location))
+                            .Should(m => m.Term("hasNonLevyContract", true))))
```

Comment:
Can you turn this into a property so we can rename it in the future

.PostFilter(pf => FilterByDeliveryModes(pf, deliveryModes) && pf.Bool(b2 => b2.Filter(f => f.Term(t => t.Field(fi => fi.HasNonLevyContract).Value(true)))));